File: C#/03. Software Technologies - юли 2017/Last Exam - 10.01.2019/SoftUni Labs/JavaScript/softunilabs/models/lab.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Lab = sequelize.define("Lab",{
        name: {
            type: Sequelize.TEXT,
        },
        capacity: {
            type: Sequelize.INTEGER,
        },
        status: {
            type: Sequelize.TEXT,
        }

    },{
        timestamps:false
    });

    return Lab;
};

File: C#/03. Software Technologies - юли 2017/Last Exam - 10.01.2019/SoftUni Labs/Java/softunilabs/src/main/java/softunilabs/entities/Lab.java
package softunilabs.entities;


import javax.persistence.*;

@Entity
@Table(name = "labs")
public class Lab {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column
    private String name;

    @Column
    private Integer capacity;

    @Column
    private String status;

    public Lab() {
    }

    public Lab(String name, Integer capacity, String status) {
        this.name = name;
        this.capacity = capacity;
        this.status = status;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getCapacity() {
        return capacity;
    }

    public void setCapacity(Integer capacity) {
        this.capacity = capacity;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}
